README_GCP.md

```diff
@@ -41,26 +41,18 @@ gcloud iam service-accounts add-iam-policy-binding opentelemetry-demo@${GCLOUD_P
     --member "serviceAccount:${GCLOUD_PROJECT}.svc.id.goog[otel-demo/opentelemetry-demo-otelcol]"
 ```
 
-Update [`gcp-config-valus.yml`](gcp-config-values.yml) to annotate the
-Kubernetes service account with your project:
+### Deploying the Helmfile
+
+Without Workload Identity, run
```

Comment:
My understanding is that we should be building for GKE Autopilot as the preferred path, which uses WI by default right? If so, we should put the workload identity steps first

Reply:
I'll address this after we get on Workload Identity Federation for GKE (used by GKE Autopilot now), instead of the legacy Workload Identity we're currently using.